### DISCUSSION TITLE
How to disable "Copy selection to clipboard" default on left double click

### DISCUSSION THREAD

**Discussion by ipekp** [flameshot-org/flameshot#2374] [Q&A] [answered]
Hi, I've been looking around in the the config file and the configuration menu of flameshot but theree seems to be no option
to disable double click.

As I'm trying to annotate or sketch on a screenshot area with my tablet this constantly closes the screenshot area and makes rather 
impossible to use.

As soon as I write something with a speed that's not slow motion, basically the screenshot overlay gets closed  😂

Any workaround this ?

Thank you

**Comment by mmahmoudian** [comment, marked as answer]
There is no way to disable, it is hardcoded and the reason was to have certain things that user cannot turn off and file a bug report. That said, I can see your point. I assume you are using Flameshot on a Windows tablet, so for the time being, you can probably increase the double click time on Windows? (I vaguely remember that this is configurable in Windows, but I haven't used Windows for about a decade)

As an *ultimate solution*, one way would be to have a tablet mode in which certain settings are changed/toggled, for instance all mouse gestures (scroll, double click, ...) are disabled.

**Reply by mmahmoudian** [reply]
@borgmanJeremy @veracioux what do you think?

**Reply by ipekp** [reply]
Hey man thanks for the answer,

I'm using flameshot on arch, by the https://wiki.archlinux.org/title/Wacom_tablet there seems to be no option to disable "double-tap" of the pencil.
That is why I've thought of that there could be an option to disable it, it would have solved the problem easily.

Changing the timers is something I could try just for the tablet input, but honestly that single option would have been great 

Thx again

**Reply by borgmanJeremy** [reply]
I think there needs to be a way to disable it . I have also accidentally activated it a few times.

**Reply by Aster89** [reply]
Which commit introduced this behavior? I'm ready to downgrade as much as it's needed to get rid of this double-tap gesture, because it makes using the tool impossible for me.

**Reply by borgmanJeremy** [reply]
You'll have to look through the commit log or PR history.

**Comment by borgmanJeremy** [comment]
#2432 Will resolve this. It makes sure only Left double click activates the copy, and adds an option to enable or disable the behavior.
